src/common/components/formikWrappers/DropdownWithFormik.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { ComponentType, ReactNode } from 'react';
+import { Dropdown, DropdownProps as HDSDropdownProps } from 'hds-react';
+import { Field, FieldProps } from 'formik';
+
+import useField from './useField';
+
+type SimulatedEvent = {
+  target: {
+    name: string;
+    value: string;
+  };
+};
+
+export type DropdownProps = Pick<
+  HDSDropdownProps,
+  'className' | 'disabled' | 'required' | 'style' | 'invalid'
+> & {
+  name: string;
+  id: string;
+  onChange: (e: SimulatedEvent) => void;
+  value: string;
+  helperText?: string | ReactNode;
+  labelText?: string | ReactNode;
+  options: Array<{ label: string; value: string }>;
+};
+
+type Props = Omit<DropdownProps, 'value' | 'onChange'> & {
```

Comment:
Personally I wouldn't make this limitation. Sometimes you may need custom `onChange` logic, or you need to provide a default value for `value` for instance.

This might have been useful when you refactored though, so I understand if it's included for that.

Reply:
This file is now removed.